sys/drv/atkbdc.c

```diff
@@ -145,6 +149,20 @@ static int atkbdc_probe(device_t *dev) {
   return 1;
 }
 
+static void evdev_init(atkbdc_state_t *atkbdc) {
```

Comment:
That init does so much that I'd like to see a brief of what is initialised

Reply:
Sure, I've added brief comments. Hope it helps.